api/utils.py

```diff
@@ -162,3 +163,41 @@ def extract_post_params(
             if value is not None:
                 data[param] = value
     return data
+
+
+def get_object_or_json_error(klass, *args, **kwargs):
+    """
+    This is exact replica of `django.shortcuts.get_object_or_404()` function
+    but instead the raised error is a `json_response_error()`.
+
+    Default message is `Not Found` and default HTTP status is `404`.
+
+    _status: int = Status code. Default 404
```

Comment:
Please opt for more words instead of `_`

Change to `status_code` instead

Reply:
The reason I put underscore prefix was to ensure that it will not conflict with any model field in the future. If you think we won't ever use `status_code` in any Model than it is OK.

Reply:
changed to `http_status_code` to be more specific